content/articles/how-to-add-support-for-spatial-data-in-your-database-using-geoalchemy-and-spatialite/index.md

```diff
@@ -0,0 +1,389 @@
+---
+layout: engineering-education
+url: /how-to-add-support-for-spatial-data-in-your-database-using-geoalchemy-and-spatialite/
+title: How To Add Support For Spatial Data in your Database Using GeoAlchemy and Spatialite
+description: This tutorial will show a step-by-step guide on how to add support to store and query spatial data for their database in their development environment.
+author: paul-asalu
+topics: []
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/how-to-build-a-clock-with-javascript-and-svg/hero.png
+    alt: Building a clock using JavaScript and SVG
+---
+
+When building certain kinds of applications, it is important to know what kind of data you'll be working with, so you can make adequate preparations to hold and process them. One such instance is when you need to work with "spatial data" - a typical example of which might be:
+<!--more-->
+
+## Table of Contents
+
+- Introduction
+- Building A Simple Web Server
+- Adding Support For Spatial Data
+- Conclusion
+
+
+## Introduction
```

Comment:
Start with `###` headers.

Make the subsequent changes

Reply:
can I use #### for sub-headings

Reply:
`###` for headings and `####` for sub-headings.

Always start with `###`

@Curiouspaul1